UX-Guide-Metadata/draft/principles/index.html

```diff
@@ -833,7 +835,7 @@ <h2>Examples</h2>
 
 						
 <section id="legal-techniques">
-					<h4> Legal considerations metadata techniques</h4>
+					<h4> Display techniques for legal considerations support</h4>
```

Comment:
Adding the word "support" here doesn't sound right. It would probably be better as "Display techniques for legal considerations"